overview/content-types/option.nix

```diff
@@ -84,11 +84,15 @@ in
           alert-update-script =
             let
               isDrv = self.type == "package";
-              optionName = lib.removePrefix "pkgs." self.default.text;
+              attrPath = lib.splitString "." self.default.text ++ [
```

Comment:
Just curious: Can we in principle get the value directly?

Reply:
If there is a way to transform a string path into an attribute set, then yes, else all we can do is check the attribute path. I don't think the former is possible, though, last time I checked.